ui/shared/RadioButtonGroup.tsx

```diff
@@ -0,0 +1,97 @@
+import { ButtonGroup, Button, Flex, Icon, useRadio, useRadioGroup, useColorModeValue } from '@chakra-ui/react';
+import type { UseRadioProps } from '@chakra-ui/react';
+import React from 'react';
+
+type RadioItemProps = {
+  title: string;
+  icon?: React.FC<React.SVGAttributes<SVGElement>>;
+  onlyIcon: false | undefined;
+} | {
+  title: string;
+  icon: React.FC<React.SVGAttributes<SVGElement>>;
+  onlyIcon: true;
+}
+
+type RadioButtonProps = UseRadioProps & RadioItemProps;
+
+const RadioButton = (props: RadioButtonProps) => {
```

Comment:
I think that will be great if we have some visual tests for `RadioButtonGroup` component